app/views/rails_admin/main/_form_boolean.html.erb

```diff
@@ -1,16 +1,14 @@
 <% if field.nullable? %>
-  <div class="btn-group" data-toggle="buttons">
-    <% {'1': true, '0': false, '': nil}.each do |text, value| %>
-      <label class="<%= [field.css_classes[value], ("active" if field.form_value == value)].compact.join(' ') %> btn btn-default">
-        <%= form.radio_button field.method_name, text, field.html_attributes.reverse_merge({ checked: field.form_value == value, required: field.required}) %>
```

Comment:
Putting an `input` element inside the label seemed not to work in Bootstrap 5, hence this structural change.